# Get a servo encoder filter's current angular position in radians. 
int32 servo_id
---
float64 angle
